images/linux/scripts/installers/nvm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  nvm.sh
+##  Desc:  Installs Nvm
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+export NVM_DIR="/opt/.nvm"
+mkdir -m 777 $NVM_DIR
```

Comment:
Why we need this magic with permissions in two places? 11 and 13 lines.
If they are necessary, please put small comment

Reply:
Done.